src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -1334,6 +1350,49 @@ uint shim::xclGetNumLiveProcesses()
     return 0;
 }
 
+int shim::xclRegRW(bool rd, uint32_t cu_index, uint32_t offset, uint32_t *datap)
+{
+    std::lock_guard<std::mutex> l(mCuMapLock);
+
+    if (cu_index >= mCuMaps.size()) {
+        std::cerr << "ERROR: xclRegRW: invalid CU index: " <<
```

Comment:
Can we use xrt_core::message::send for all cerr calls?

Reply:
> 
> 
> Can we use xrt_core::message::send for all cerr calls?

Will do!